src/components/reusableComponents/profile/RoleIntro.tsx

```diff
@@ -1,32 +1,43 @@
 import './styles.css';
-
-import colors from '../../../consts/colorPallete';
-import useCurrentTheme from '../../../consts/theme';
-// import { selectCurrentUserRole, selectUserSessionState, UserRole } from '../../../redux/reducers/user-state';
-
-import { AddCircleOutline, RemoveCircleOutline, SelfImprovement } from '@mui/icons-material';
-import { useEffect, useState } from 'react';
-import { principalCV, ClarityValue, listCV, cvToJSON } from '@stacks/transactions';
 import { useAppSelector } from '../../../redux/store';
-import { selectCurrentUserRole, selectUserSessionState, UserRole } from '../../../redux/reducers/user-state';
-import { ReadOnlyAllDataWaitingMiners, readOnlyGetRemainingBlocksJoin } from '../../../consts/readOnly';
+import { selectCurrentUserRole, UserRole } from '../../../redux/reducers/user-state';
 import RoleIntroMiner from './RoleIntroMiner';
 import RoleIntroWaiting from './RoleIntroWaiting';
 import RoleIntroPending from './RoleIntroPending';
 import RoleIntroNormalUser from './RoleIntroNormalUser';
 
 interface IRoleIntro {
   currentRole: string;
+  positiveVotes: number | null;
+  positiveVotesThreshold: number | null;
+  negativeVotes: number | null;
+  negativeVotesThreshold: number | null;
+  blocksLeftUntilJoin: number | null;
 }
 
-const RoleIntro = ({ currentRole }: IRoleIntro) => {
+const RoleIntro = ({
```

Comment:
this refers to the banner info on Profile. maybe rename this variable to something more related?